compose/snippets/src/main/java/com/example/compose/snippets/designsystems/Material2Snippets.kt

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-@file:Suppress("unused")
+@file:Suppress("unused", "DEPRECATION_ERROR")
```

Comment:
if something was deprecated here, i think we should check what it is and try to fix the DAC page.

it's ok to add this for now, but we should at least create a bug internally

Reply:
I don't think we want to update the DAC page's snippet yet since 1.7.0 is still in alpha. We'll want to do that once 1.7.0 is stable (and the change would have to do be done on `main`)